### PR TITLE
readme: update params and keys, add little note

### PR DESCRIPTION
**Pull request by masterSplinter01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,9 +221,11 @@ For example:
 $ neofs-cli -r 192.168.130.72:8080 -k 6PYLKJhiSub5imt6WCVy6Quxtd9xu176omev1vWYovzkAQCTSQabAAQXii object put --file cat.png --cid DPL2tpRiuDNmoTj5KZjD1nzDuCS8tVcxa7hsvSLDWpVM --attributes img_type=cat,my_attr=cute
 ```
 
-
 ### Downloading
 
+> If you want to download from/upload to a private or public-read container, 
+make sure you've executed a gateway with your private key (see [Keys](#Keys) and [Authentication](#Authentication)).
```
Comment (roman-khimov) [inline README.md:227]: I'm not sure we need this, we do have `Authentication` section.

File: README.md [hunk 2]
```diff
@@ -86,15 +86,15 @@ This command will make gateway use 192.168.130.72 for 90% of requests and
 
 By default gateway autogenerates key pair it will use for NeoFS requests. If
 for some reason you need to have static keys you can pass them via `--key` 
-(or `-k`) parameter. The key can be a path to private key file (as raw bytes), 
+parameter. The key can be a path to private key file (as raw bytes), 
 a hex string or (unencrypted) WIF string:
 ```
-$ neofs-http-gw -p $NEOFS_NODE -k $KEY
+$ neofs-http-gw -p $NEOFS_NODE --key $KEY
```
Comment (roman-khimov) [inline README.md:92]: Why don't we fix `-k`?

File: README.md (hunk 1 above)
Comment (masterSplinter01) [reply README.md:227]: IMO, Authentication section doesn't make clear how to pass a key for uploading/downloading files to private containers. At least, for people who trying to do that for the first time. Maybe Keys or Authentication sections should be moved somewhere in the file, but I don't know where.

File: README.md (hunk 2 above)
Comment (masterSplinter01) [reply README.md:92]: Fair enough.

File: README.md (hunk 1 above)
Comment (roman-khimov) [reply README.md:227]: You can extend them, but I don't think it's worth moving them.

File: README.md [hunk 3]
```diff
@@ -221,7 +221,6 @@ For example:
 $ neofs-cli -r 192.168.130.72:8080 -k 6PYLKJhiSub5imt6WCVy6Quxtd9xu176omev1vWYovzkAQCTSQabAAQXii object put --file cat.png --cid DPL2tpRiuDNmoTj5KZjD1nzDuCS8tVcxa7hsvSLDWpVM --attributes img_type=cat,my_attr=cute
 ```
 
-
```
Comment (roman-khimov) [inline README.md:224]: Not really needed.

Comment (roman-khimov) [reply README.md:224]: Still there.

Comment (masterSplinter01) [reply README.md]: Fixed.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**